include/SW_Sky.h

```diff
@@ -28,8 +28,8 @@ extern "C" {
 /*             Global Function Declarations            */
 /* --------------------------------------------------- */
 void SW_SKY_read(char *InFiles[], SW_SKY* SW_Sky, LOG_INFO* LogInfo);
-void SW_SKY_new_year(SW_MODEL* SW_Model, RealD snow_density[MAX_MONTHS],
-					 RealD snow_density_daily[MAX_MONTHS]);
+void SW_SKY_new_year(SW_MODEL* SW_Model, TimeInt startyr,
+	RealD snow_density[MAX_MONTHS], RealD snow_density_daily[MAX_MONTHS]);

```

Comment:
Now that SW_MODEL regained time, we can collapse `SW_SKY_new_year(SW_MODEL* SW_Model, TimeInt startyr,...` again